README.md

```diff
@@ -1,201 +1,37 @@
-# Hathor Network
+# Hathor core
 
-[![Mainnet](https://img.shields.io/badge/mainnet-live-success)](https://explorer.hathor.network/)
-[![Version](https://img.shields.io/github/v/release/HathorNetwork/hathor-core)](https://github.com/HathorNetwork/hathor-core/releases/latest)
-[![Testing](https://img.shields.io/github/actions/workflow/status/HathorNetwork/hathor-core/main.yml?branch=master&label=tests&logo=github)](https://github.com/HathorNetwork/hathor-core/actions?query=workflow%3Atests+branch%3Amaster)
-[![Docker](https://img.shields.io/github/actions/workflow/status/HathorNetwork/hathor-core/docker.yml?branch=master&label=build&logo=docker)](https://hub.docker.com/repository/docker/hathornetwork/hathor-core)
 [![Codecov](https://img.shields.io/codecov/c/github/HathorNetwork/hathor-core?logo=codecov)](https://codecov.io/gh/hathornetwork/hathor-core)
-[![Discord](https://img.shields.io/discord/566500848570466316?logo=discord)](https://discord.com/invite/35mFEhk)
-[![License](https://img.shields.io/github/license/HathorNetwork/hathor-core)](./LICENSE.txt)
 
-## Running a full-node
+## Description
 
-**Disclaimer**
+**Hathor core** is the main software component of a Hathor full node. Once a computer system runs Hathor core and connects to Hathor Network it becomes a Hathor full node.
 
-At the moment, our mainnet is running on a whitelist basis. This means only authorized nodes will be able to connect. For testing purposes, you can connect to the testnet (using the `--testnet` parameter). If you want to connect to the mainnet, you have to [use a peer-id](#using-a-peer-id) and send this id to a team member. You can get in touch with us through [our channels](https://hathor.network/community/), preferrably Discord.
+**Hathor full node** is the computer system unit that is a peer in Hathor Network. Hathor full node is the primary building block that comprises Hathor Network.
 
-### Using Docker
+## Operation and usage
 
-The easiest way to run a full-node is to use our Docker image. If you don't have Docker installed, check out [this
-link](https://docs.docker.com/install/). So, just run:
+To know how to operate and use Hathor core, see [Hathor full node at Hathor docs — official technical documentation of Hathor](https://docs.hathor.network/pathways/components/full-node).
```

Comment:
The official technical documentation seems to be outdated:

- latest stable version is v0.58.0 and latest "unstable" version is v0.59.0-rc.1
- deprecated versions, we definitely don't support running too old versions, I can't recall exactly from what version but there certainly is a cutoff.
- not sure what is the best way to keep that document up to date, but maybe we should update that document before we start pointing to it

Reply:
- The [Hathor full node](https://docs.hathor.network/pathways/components/full-node) page of the official technical documentation is already updated, indicating that the latest stable version is: v0.58.0.
- As for unstable versions, these should not be reported in the official documentation.
- My commitment will be to keep this page updated with the same version marked with the latest tag on GitHub. So whenever a new version is marked as stable on GitHub, I will update the page.
- Regarding deprecated versions: it is important to mark, if possible, which versions are deprecated. If you are not sure, I suggest we use as a cutoff point the most recent version that you know no node in the network is operating with.
- Regarding updating this page beforehand and pointing to it: note that the information we currently find in the README is much more outdated than what is in the official documentation.
- Because of this, I do not see it as a problem to update the README pointing there.
- Moreover, all the hathor core operation documentation is being updated: how to install, how to update, etc.

Reply:
Sounds good, I'm very positive about moving all of that away from the README. What I had in mind was something that could go either of these ways:

- Instead of having the version on the document we could just point to the Github releases page which always has the latest version
- Make the CI update the documentation (it would be simpler if the documentation was generated)
- Include "notify the documentation maintainer" to the release process; this is the closest to what you've proposed but I think it would be helpful to have this step documented on the release process

Reply:
Change made:
I created a new react component to our docs website.
Now, the latest version is not "hardcoded" as text anymore.
The react component fetches latest version at `https://github.com/HathorNetwork/<name_of_repo>/releases/latest` when client side renders the webpage.

Thus, latest version presented in the docs webpages of core, headless and others will always be the latest one.